package.json

```diff
@@ -178,6 +179,10 @@
                     "command": "aws.searchSchemaPerRegistry",
                     "when": "false"
                 },
+                {
+                    "command": "aws.submitFeedback",
+                    "when": "false"
```

Comment:
Just making sure, this excludes the command from the command palette? I feel like it belongs there as a secondary entrypoint.

Reply:
Agreed, added it back.